### DISCUSSION TITLE
Be able to bypass approval when creating via a factory

### DISCUSSION THREAD

**Discussion by Temepest74** [cjmellor/approval#63] [Ideas]
Idk fi I do it wrong, but I cannot do it.
I tried 
```
            $createdModel = User::factory()->withoutApproval()->create();
```
which gives me `call to undefined method`
and this which simply does not work
```
            $createdModel = User::factory()->create()->withoutApproval();
```
And I think a toggle in the factory or this function being implemented in the factory will help a lot. Or maybe it is related to #26 ? Which you cant really solve as you did when you have seeders.

While I could technically approve each model individually as a last resort, it will create records in the approvals table for no reason, and this will get troublesome when using count to seed many models

**Comment by cjmellor** [comment]
Merged in #64
